scripts/deploy.js

```diff
@@ -0,0 +1,13 @@
+const exec = require('child_process').exec;
+
+exec('npm publish', (error, stdout, stderr) => {
```

Comment:
We'd have to get Yarn into the shipit environment.  I was able to hack that together as part of a deployment, but it wasn't pretty (and without a cache dir, it seemed slower than npm).

My thought was to not worry about Yarn until we got to the point of changing the shipit-engine code, but I can start that process now if you think it'll speed things along?